numaprom/_config.py

```diff
@@ -34,5 +36,31 @@ class AppConf:
 
 
 @dataclass
-class NumapromConf:
+class DataConf:
     configs: List[AppConf]
+
+
+@dataclass
+class RedisConf:
+    host: str
+    port: int
+    expiry: int = 300
+    master_name: str = "mymaster"
+
+
+@dataclass
+class PrometheusConf:
+    server: str
+    pushgateway: str
+
+
+@dataclass
+class RegistryConf:
+    tracking_uri: str
```

Comment:
It is fine to keep this here for now, but should move to numalogic soon

Reply:
I think registry itself should be moved to a new repository, we can move this when we move registry